app/src/sharedTest/java/org/oppia/android/app/testing/PlatformParameterIntegrationTest.kt

```diff
@@ -119,42 +105,20 @@ class PlatformParameterIntegrationTest {
   @Test
   fun testSplashTestActivity_readEmptyDatabase_checkWelcomeMsgIsInvisibleByDefault() {
     launch(SplashTestActivity::class.java).use {
-      onView(withText(SplashTestActivity.WELCOME_MSG)).check(doesNotExist())
+      testCoroutineDispatchers.runCurrent()
+      assertThat(ShadowToast.getLatestToast()).isNull()
     }
   }
 
-  @RunOn(TestPlatform.ESPRESSO)
   @Test
   fun testSplashTestActivity_updateEmptyDatabase_readDatabaseValues_checkWelcomeMsgIsVisible() {
     platformParameterController.updatePlatformParameterDatabase(mockPlatformParameterList)
     testCoroutineDispatchers.runCurrent()
 
     launch(SplashTestActivity::class.java).use {
-      onView(withText(SplashTestActivity.WELCOME_MSG))
-        .inRoot(getToastMatcher())
-        .check(matches(isDisplayed()))
-    }
-  }
-
-  // Returns a [TypeSafeMatcher] which checks for any [Toast] message in the screen.
-  private fun getToastMatcher(): TypeSafeMatcher<Root?> {
-    return object : TypeSafeMatcher<Root?>() {
-      override fun describeTo(description: Description?) {
-        description?.appendText("is toast")
-      }
-
-      override fun matchesSafely(item: Root?): Boolean {
-        val type: Int? = item?.windowLayoutParams?.get()?.type
-        if (type == WindowManager.LayoutParams.TYPE_TOAST) {
-          val windowToken: IBinder = item.decorView.windowToken
-          val appToken: IBinder = item.decorView.applicationWindowToken
-          if (windowToken === appToken) {
-            // Means this window isn't contained by any other windows.
-            return true
-          }
-        }
-        return false
-      }
+      testCoroutineDispatchers.runCurrent()
+      assertThat(ShadowToast.getLatestToast()).isNotNull()
+      assertThat(ShadowToast.getTextOfLatestToast()).isEqualTo(SplashTestActivity.WELCOME_MSG)
```

Comment:
Is this the only way to verify the toast? If possible I'd also add a check that the toast view is fully visible.

Reply:
@anandwana001 do you have any suggestions on this?

Reply:
I need to check if decor view works with both espresso and robolectric or not.

Reply:
From an offline conversation we learned that Shadows are not compatible with Espresso so we won't be verifying any UI elements with these tests. This comment is not applicable anymore.